source/index.html

```diff
@@ -1,12 +1,208 @@
 <!DOCTYPE html>
-<html lang="ru">
+<html class="page" lang="ru">
   <head>
     <meta charset="utf-8">
     <title>HTML Academy: Мишка</title>
   </head>
-  <body>
-
-    <p>Репозиторий создан для обучения на профессиональном онлайн‑курсе «<a href="https://htmlacademy.ru/intensive/adaptive">HTML и CSS. Адаптивная вёрстка и автоматизация</a>».</p>
-
+  <body class="page__body">
+    <header class="page-header">
+      <a class="page-header__logo" href="#">
+        <svg width="86" height="35" fill="none" xmlns="http://www.w3.org/2000/svg" aria-hidden="true" focusable="false" aria-label="Переход на главную страницу магазина «Мишка»." role="img">
```

Comment:
```suggestion
        <svg width="86" height="35" fill="none" xmlns="http://www.w3.org/2000/svg" aria-label="Переход на главную страницу магазина «Мишка»." role="img">
```

тут однозначно не скрывать, тк это главный первый таб для фокуса